### ISSUE TITLE
Multiline indentation should be accepted when a method is an argument to another method

### ISSUE THREAD

**Issue by halo** [bbatsov/rubocop#3969]
The `Style/MultilineMethodCallIndentation` cop does not like this:

```ruby
@users = authorize User.includes(:blogs)
                       .where(name: 'Bob')
                       .order(:name)
```

and suggests that I use this instead:

```ruby
@users = authorize User.includes(:blogs)
         .where(name: 'Bob')
         .order(:name)
```

Is this intended behavior? That kind of syntax is common with the [pundit](https://github.com/elabs/pundit) gem.

(RuboCop is fine with the following though, notice the paranthesis between `authorize` and `User`)

```ruby
@users = authorize(User.includes(:blogs)
                       .where(name: 'Bob')
                       .order(:name))

```

This issue [appears not to be reported](https://github.com/bbatsov/rubocop/search?utf8=✓&q=multiline+is%3Aopen&type=Issues) and I've been testing this against [latest master](https://github.com/bbatsov/rubocop/commit/2e70d6fc45c29a71c7ae922c9599b69cc988dd9a). I'd be glad to provide a PR if you believe this is a bug.

--------

## Expected behavior

I expected RuboCop not to complain about the indentation because the method calls are aligned with the original receiver `User`.

## Actual behavior

RuboCop complains about the indentation and suggests (and auto-corrects) to have the chained methods be aligned with `authorize` instead.

## Steps to reproduce the problem

[Here](https://github.com/halo/rubocop/pull/1/files) is an example of two failing specs that I think should pass instead.

## RuboCop version

```
$ rubocop -V
0.47.1 (using Parser 2.3.3.1, running on ruby 2.3.3 x86_64-darwin16)
```

## Tell the RuboCop-Team just how awesome they are

You are awesome! The Ruby community is so propelled by you.

**Comment by bbatsov** [comment]
> Is this intended behavior? That kind of syntax is common with the pundit gem.

That's a bug for sure.

**Comment by halo** [comment]
So, after 2 hours of [puts debuggering](http://tenderlovemaking.com/2016/02/05/i-am-a-puts-debuggerer.html) along [these lines](https://github.com/bbatsov/rubocop/blob/0293ef2637913280ee7670511a333084df5ecb39/lib/rubocop/cop/mixin/multiline_expression_indentation.rb#L15-L17) I feel I'm not going so solve this bug in the near future. Sorry, I guess I'm just not familiar enough with the terminology.

If it gets over me, I'll give it another try. But until then I'll leave it to the enlightened :) Thank you for your confirmation that this is a bug.

**Comment by jfelchner** [comment]
I was looking through all of the issues because someone said there may be a gotcha in my PR (I haven't found one yet), but [my PR](https://github.com/bbatsov/rubocop/pull/4134) should fix this.

**Comment by jfelchner** [comment]
Actually nm, I'm sorry for the false hope.  It looks like you're using `aligned` and not `indented_relative_to_receiver`.

**Comment by halo** [comment]
@jonas054 @bbatsov Thank you indeed for your hard work on this. 🥇
